pages/common/passwd.md

```diff
@@ -2,6 +2,10 @@
 
 > Passwd is a tool used to change a user's password.
 
+- Change the password interactively:
```

Comment:
```suggestion
- Change the password of the current user interactively:
```
Let's specify that's the password of the *current* user.

Reply:
It makes sense. Thanks for the suggestion.